03LessonThree/in-class-project/daily-planet.html

```diff
@@ -1,10 +1,59 @@
+                    more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.</p>
+                <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen
+                    book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and
+                    more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum. </p>
+                <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen
+                    book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and
+                    more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.</p>
+            </article>
+        </section>
+    </main>
+    <aside>
+        <a href="insightly.com">
+            <img src="https://aka-cdn.adtechus.com/images/412/Ad0St1Sz529Sq0V0Id24953244.png" />
+        </a>
+    </aside>
+    <footer>
+        <table>
+            <tr>
+                <td>
+                    <a href="https://www.facebook.com/sharer/sharer.php?u=http://interactives.dallasnews.com/2016/texas-olympians/"> <img src="https://www.facebookbrand.com/img/fb-art.jpg" </td>
+                        <td>
+                            <a href="https://twitter.com/intent/tweet?text=Keep%20up%20with%20all%20of%20the%20Texas%20olympians%20with%20our%202016%20Summer%20Olympics%20tracker%20%7C%20@SportsDayDFW&url=http://interactives.dallasnews.com/2016/texas-olympians/">
+                            <img src="https://cdn3.iconfinder.com/data/icons/social-icons-5/607/Twitterbird.png"> </td>
+                        <td>
+                            <a href="mailto:test@test.com"> <img src="http://www.freeiconspng.com/uploads/email-icon-23.png" </td>
+                                <td>
+                                    <a href="https://www.instagram.com/?hl=en"> <img src="http://3835642c2693476aa717-d4b78efce91b9730bcca725cf9bb0b37.r51.cf1.rackcdn.com/Instagram_App_Large_May2016_200.png" </td>
+        </table>
+    </footer>
+</body>
+
+</html>

```

Comment:
Great job! 

You are missing one of the project requirements - Add copyright and company name to footer. If you can add and recommit, I can bump your grade up 5 points.